srcpkgs/niri/template

```diff
@@ -13,7 +13,10 @@ license="GPL-3.0-or-later"
 homepage="https://github.com/YaLTeR/niri"
 changelog="https://github.com/YaLTeR/niri/releases"
 distfiles="https://github.com/YaLTeR/niri/archive/refs/tags/v${version}.tar.gz"
-checksum=d8854830436a87215b0bc6a60b6d43f350d927a03a2798c75f0fbda228bac8d3
+checksum=86b89bcfc3fc6a8ed81f9e3f0ac7a29bd30267515efb2c19e1e0bc2ccd67b649
+
+export NIRI_BUILD_COMMIT="e35c630"
```

Comment:
would be better to use something like `v$version`, as

1. hardcoding things like that is bad and will be forgotten at some point
2. we build from the tag, which is a reference to the commit

Reply:
This variable is in accordance with Niri's own packaging guidelines

Reply:
upstreams do stuff like this all the time. if they aren't leaving the commit in their release tarballs somehow (like a file), we shouldn't need to hard-code it. we aren't building some random commit, we're building the release, so the commit information is redundant.

Reply:
Understood and agreed that it is redundant, however if it is expected by upstream, I'm not sure I really think it is worth it to break that expectation downstream is all (i.e. for bug reports and such)

Reply:
If upstream wants this information to be accurate, they should add it to their release artifacts. they already generate them to vendor cargo dependencies.

Reply:
Fair point, I might poke upstream to add that, but then tbh I think it is best to just not add any environment variable, since it will report `(unknown commit)` by default, which is more meaningful here